secret-service/pkg/backend/secretbackend_k8s_test.go

```diff
@@ -422,6 +447,23 @@ func TestUpdateSecret_SecretNotFound(t *testing.T) {
 	assert.Equal(t, ErrSecretNotFound, err)
 }
 
+func TestUpdateSecret_ScopeNotFound(t *testing.T) {
```

Comment:
Does it makes sense to to create a similar unit test also for creating a secret ?

Reply:
ah sorry, It already exists, that's why I was missing it :)